src/openzaak/conf/includes/base.py

```diff
@@ -171,6 +171,9 @@
     ),
     auto_display_default=False,
 )
+if not OPENZAAK_DOMAIN:
```

Comment:
I think we should probably deprecate OPENZAAK_DOMAIN and rely on SITE_DOMAIN instead, add a fallback that's the other way around to ensure that the current usage of OPENZAAK_DOMAIN still works, and then in the next major release we can drop it